tests/test_actinia_mapset_management.py

```diff
@@ -81,6 +96,24 @@ def setup_class(cls):
     def teardown_class(cls):
         cls.mock_get_patcher.stop()
         cls.mock_post_patcher.stop()
+        cls.mock_delete_patcher.stop()
+        if NEW_LOCATION_NAME in cls.testactinia.locations:
+            cls.mock_delete_mapset()
+            cls.testactinia.locations[NEW_LOCATION_NAME].delete_mapset(
+                NEW_MAPSET_NAME
+            )
+
+    @classmethod
```

Comment:
Is the annotation `@classmethod` needed here and some lines below?

Reply:
I removed one unnecessary function, and I'm not 100% sure if the `mock_delete_mapset` method needs to be a classmethod but it is consistent with how the `mock_delete_location` is set up in `tests/test_actinia_location_management.py`. @anikaweinmann?

Reply:
I think it is okay